regression-tests.dnsdist/dnsdisttests.py

```diff
@@ -153,22 +190,7 @@ def setUpClass(cls):
 
     @classmethod
     def tearDownClass(cls):
-        cls._sock.close()
-        if 'DNSDIST_FAST_TESTS' in os.environ:
-            delay = 0.1
-        else:
-            delay = 1.0
-        if cls._dnsdist:
-            cls._dnsdist.terminate()
-            if cls._dnsdist.poll() is None:
-                time.sleep(delay)
-                if cls._dnsdist.poll() is None:
-                    cls._dnsdist.kill()
-                cls._dnsdist.wait()
-
-        # tell the background threads to stop, if any
-        for backgroundThread in cls._backgroundThreads:
-          cls._backgroundThreads[backgroundThread] = False
+        cls.killProcess(cls._dnsdist)
```

Comment:
Have the `cls._sock.close()` line and the cleaning of the background threads been removed on purpose?

Reply:
Oops